apps/tangle-dapp/constants/bridge.ts

```diff
@@ -34,6 +34,28 @@ export const BRIDGE_SUPPORTED_TOKENS: Record<BridgeTokenId, BridgeTokenType> = {
         '0xA9F30c6B5E7996D1bAd51D213277c30750bcBB36',
     },
   },
+  // TODO: remove this (this is for testing Hyperlane only)
+  WETH: {
+    id: 'WETH',
+    symbol: 'WETH',
+    existentialDeposit: {},
+    destChainTransactionFee: {},
+    decimals: {
+      default: 18,
+    },
+    erc20TokenContractAddress: {
+      [PresetTypedChainId.Sepolia]:
+        '0x0F4C1d951295Fe17c1514eB3020dFA6EedAd0137',
+      [PresetTypedChainId.Holesky]:
+        '0x94373a4919B3240D86eA41593D5eBa789FEF3848',
+    },
+    hyperlaneRouteContractAddress: {
+      [PresetTypedChainId.Sepolia]:
+        '0x9DB8ebb2666E2e9f6864A82272199632eE45d182',
+      [PresetTypedChainId.Holesky]:
+        '0x9DB8ebb2666E2e9f6864A82272199632eE45d182',
+    },
+  },
```

Comment:
Idea: We could leave it in and then have a condition like `if !IS_PRODUCTION_ENV` then hide, so that we can test Hyperlane later on without needing to re-add it

Reply:
I will update this but eventually we will remove it when the routes for Tangle's assets are ready

Reply:
On second thought, I don't think we need to add this.

Adding env check requires adding `IS_PRODUCTION_ENV` in many files which is time-consuming for now and for the future when we decide to remove it (which we definitely will).